src/lamplib/src/genny/tasks/yaml_linter.py

```diff
@@ -6,11 +6,14 @@
 import structlog
 import yamllint.cli
 from genny.tasks.yaml_linter_constants import (
+    GRANDFATHERED_WORKLOADS_OWNERS_NOT_IN_MOTHRA,
     REQUIRED_FIELDS,
     GRANDFATHERED_WORKLOADS_WITHOUT_KEYWORDS,
 )
+from genny.tasks.mothra_service import MothraService
 
 SLOG = structlog.get_logger(__name__)
+mothra_service = MothraService()
```

Comment:
Can we do this in main instead? I don't like that importing this module is going to eventually do a `git clone`.